models/settings.go

```diff
@@ -58,6 +58,11 @@ type Settings struct {
 		// Security Threat Tool enabled
 		STTEnabled bool `json:"stt_enabled"`
 	} `json:"sass"`
+
+	// DBaaS config options
+	DBaaS struct {
+		DBaaSEnabled bool `json:"dbaas_enabled"`
```

Comment:
Why not just `enabled` as telemetry above?

Reply:
Because it's made like STT above

Reply:
It is not like STT: STT setting is in SaaS group, not STT group. Here we have DBaaS setting in DBaaS group

Reply:
Aha, good point, ok